stdlib/strings/strings.flux

```diff
@@ -1,39 +1,780 @@
+// The strings package provides functions to manipulate UTF-8 encoded strings.
 package strings
 
 
-// Transformation functions
+// Transformation functions:
+//
+// The strings.title() function converts a string to title case.
```

Comment:
Can you make these lines start with  the name of the builtin/function like the spec defines? i.e. `title converts a string to title case`?

I am getting better a reviewing these as I see more of them. Thanks for being patient with all the iterations.